config/webpack/webpack.config.js

```diff
@@ -95,16 +96,19 @@ module.exports = {
       VERSION: JSON.stringify(packageFile.version),
     }),
     new MomentLocalesPlugin(),
+    new CleanWebpackPlugin(),
     // new BundleAnalyzerPlugin({
     //   defaultSizes: 'gzip',
     //   excludeAssets: '.*\.hot-update\.js',
     // }),
   ],
   optimization: {
-    moduleIds: 'hashed',
+    runtimeChunk: 'single',
     splitChunks: {
       chunks: 'all',
+      minSize: 1000 * 500,
```

Comment:
just curious, why 500k?

Reply:
Trial and error testing so that quill wouldn't be loaded in more than one bundle as it's a fairly large package. I followed this article and it basically said use trial and error here https://blog.logrocket.com/guide-performance-optimization-webpack/